src/js/components/sidebar.jsx

```diff
@@ -327,6 +384,21 @@ export default class SideBar extends React.Component {
     browser.tabs.remove(this._getSelectedTabIds());
   }
 
+  _reloadSelected() {
+    let selectedTabIds = this._getSelectedTabIds();
+
+    for (let id of selectedTabIds) {
+      // reload each selected tab.
+      browser.tabs.reload(id);
+    }
+  }
+
+  async _moveSelectedToNewWindow() {
+    let selectedTabs = this._getSelectedTabIds();
+    let newWindow = await browser.windows.create({tabId: selectedTabs[0]});
+    browser.tabs.move(selectedTabs, {windowId: newWindow.id, index: 0});
```

Comment:
Isn't `selectedTabs[0]` already in the new window? Shouldn't we be moving the range `selectedTabs[1..]` ?